### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by iBug**
We're no longer using ChatExchange as a submodule, right?

### CODE REVIEW

**Comment by angussidney** [comment]
'Credidential' should be 'credidential'

Also bonus marks if you can add a copy of the old setp.sh file to the repo so that we don't need to add our own 👍

**Comment by iBug** [comment]
@angussidney Please don't taunt me for my high typo rate (I code on mobile)... and yeah, I'll add an example file which should not be tracked by Git after modification.

File: setp.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# This file will not be tracked by git as it's declared in .gitignore
```
Comment (tripleee) [inline setp.sh]: This is obviously not true.

Comment (iBug) [reply setp.sh]: First, I use GH web editor. Second, there's `git add --force` that force stages a file even if it's in .gitignore.

**Pull request merged**